src/content/en/agencies/_index.yaml

```diff
@@ -0,0 +1,95 @@
+project_path: /web/_project.yaml
+book_path: /web/_book.yaml
+title: Find a qualified web dev agency to help you build optimized web apps
+description: >
+  These agencies have completed our advanced Web Apps Training for Agencies -
+  an intense course covering the Developing PWAs syllabus and hands-on codelabs.
+
+landing_page:
+  body_class: web-landing-page web-landing-page--agencies
+  custom_css_path: /web/agencies/css/web-agencies-styles.css
+  rows:
+
+  # INTRO
+  - classname: web-hero devsite-landing-row-50 devsite-landing-row-large-headings devsite-landing-row-no-image-background
+    items:
+    - heading: Web Apps Training for Agencies
+      description: >
+        An intense two-day course for web development agencies that covers the
+        <a href="https://developers.google.com/training/courses/progressive-web-apps">
```

Comment:
Don't include the domain, if they're on the .cn site, it won't work. Simplify this to `href="/training/courses/..."`